pkg/machine/qemu/machine.go

```diff
@@ -1735,3 +1735,11 @@ func isProcessAlive(pid int) bool {
 func (p *Provider) VMType() string {
 	return vmtype
 }
+
+func isRootfull() bool {
```

Comment:
Nit: We spell Rootfull-> Rootful, only one `l`

Reply:
Changed it and forced pushed.